sw/airborne/arch/chibios/subsystems/radio_control/ppm_arch.c

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
I remember we were having issues with PPM on Lisa M because the ICU driver in ChibiOS 2.6 didn't support Timer Channel 3 which was in this case the channel used for PPM rx. Do you know if this is fixed in new ChibiOS?